### ISSUE TITLE
Dont print info by default

### ISSUE THREAD

**Issue by KristofferC** [JuliaFEM/AbaqusReader.jl#45]
It would be nice if there was a `verbose` option to turn of the info-printing but have it be quiet by default.

**Comment by ahojukka5** [comment]
What about if we change those to debug? In a bigger picture, we should have some kind of logging system so that we can write messages to log files. Also, it would be nice to control the number of logging messages based on that is user executing functions interactively in REPL or running them from the script file.

**Comment by KristofferC** [comment]
Debug probably also works, but I am not very familiar with the Base Logging system.

**Comment by benzwick** [comment]
I agree that changing these messages to debug is a good idea. Maybe leave the warnings as they are though (or change those to info if they are not very important).

If a user wants to write messages to log files they can easily substitute the base logging with e.g. [Memento](https://invenia.github.io/Memento.jl/latest/#Quick-Start-1). This means we can continue to use base logging and let users decide what they want to do with their logs.
